capstone/src/main/webapp/group.html

```diff
@@ -3,44 +3,52 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link
       href="http://fonts.googleapis.com/css?family=Roboto"
       rel="stylesheet"
       type="text/css"
     />
     <link rel="stylesheet" href="styles/group.css"></link>
-    <script src="scripts/poll.js"></script>
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <script src="scripts/post.js"></script>
+    <script src="scripts/poll.js"></script>
+    <script src="https://unpkg.com/ionicons@5.1.2/dist/ionicons.js"></script>
     <title>Group</title>
   </head>
 
-  <body onload="getPollOptions()">
-    <div class="poll-container">
-      <div class="inner-poll-container">
-          <div class="section-text">
-              Poll
-          </div>
-          <div id="options-container"></div>
-          </div>
-        <input id="input-box" name="input-box" placeholder="Add a suggestion" onkeydown="if (event.keyCode == 13) addPollOption()"></input>
+  <body onload="init()">
+    
+    <div class="group-content">
+
+      <div class="group-header">
+        <div class="group-name-banner">
+          <span id="group-name" class="align-vertical">The 3 Musketeers + Others</span>
+        </div>
       </div>
-    </div>
```

Comment:
This is probably because y'all are working on the same file so merging and stuff with Git gets tricky but when you merge this into master Lucy's code won't be deleted, right?

Reply:
Nope, removed it for the time being so I could work on my features. Will pull from master and add back before pushing!